packages/web3-shared/base/src/specs/index.ts

```diff
@@ -887,15 +893,17 @@ export interface Hub<ChainId, SchemaType, GasOption, Web3HubOptions = HubOptions
         initial?: Web3HubOptions,
     ) => Promise<Pageable<NonFungibleToken<ChainId, SchemaType>>>
     /** Get all approved fungible tokens of given account. */
-    getApprovedFungibleTokens?: (
+    getApprovedFungibleTokenSpenders?: (
```

Comment:
Use `getApprovedFungibleTokens` and `getApprovedNonFungibleTokens` in hub.

Reply:
I just renamed it.